docs/api.swagger.yaml

```diff
@@ -1403,8 +1420,8 @@ paths:
       tags:
         - warden
         - groups
-      summary: Find group IDs by member
-      operationId: findGroupsByMember
+      summary: Find group IDs
```

Comment:
In fact, we're receiving the full group payload (no longer only their IDs). This isn't your fault, but mine since I forgot to update the header title. But since you're already at it, maybe you could change that too ... :)

Reply:
This applies to the 0.10.x branch - for the 0.9.x branch this is correct.